images/scripts/rhel-8-1-distropkg.install

```diff
@@ -2,4 +2,7 @@
 
 set -e
 
+# remove cockpit distro packages, testing with upstream master
+rpm --erase --verbose cockpit-dashboard
```

Comment:
This shouldn't be necessary. You changed rhel.setup to only install cockpit-dashboard on rhel-7* and centos-7, not on the RHEL images. Dito for the other rhel-8-* scripts.